src/activities/components/ParticipantTypesEdit.vue

```diff
@@ -0,0 +1,471 @@
+<template>
+  <div>
+    <div class="row">
+      <QToggle
+        v-if="isFeatureEnabled || isUsingAdvanced"
+        v-model="advancedMode"
+        label="Use participant types"
```

Comment:
i18n

Reply:
> "use participant types" in ActivitySeriesEdit could be "use multiple participant types"

I went through various different wordings, ... it's useful to enable even if you don't have multiple, as it allows limiting by role...